services/httpd/response_logger.go

```diff
@@ -101,21 +101,46 @@ func buildLogLine(l *responseLogger, r *http.Request, start time.Time) string {
 
 	userAgent := r.UserAgent()
 
-	return fmt.Sprintf(`%s - %s [%s] "%s %s %s" %s %s "%s" "%s" %s %d`,
-		host,
-		detect(username, "-"),
-		start.Format("02/Jan/2006:15:04:05 -0700"),
-		r.Method,
-		uri,
-		r.Proto,
-		detect(strconv.Itoa(l.Status()), "-"),
-		strconv.Itoa(l.Size()),
-		detect(referer, "-"),
-		detect(userAgent, "-"),
-		r.Header.Get("Request-Id"),
-		// response time, report in microseconds because this is consistent
-		// with apache's %D parameter in mod_log_config
-		int64(time.Since(start)/time.Microsecond))
+	if r.Method == "POST" {
+		vk := make([]string, 0, len(r.PostForm))
+		for k, v := range r.PostForm {
+			vk = append(vk, fmt.Sprintf("\"%s\": \"%s\"", k, v))
```

Comment:
`vk` is not a great variable name.

Also are we at all worried about someone sending absolutely huge queries that are expensive to log / hit log line limits?

Reply:
I am not concerned with overrunning line lengths. We have no log line limits I have found; do you know of any? And if a customer wants to send a huge query, we should log it; this logging can be disabled if desired.